documentation/plain_rules.md

```diff
@@ -139,6 +139,8 @@ This substitution can be used in the following fields:
 -   `template` for `twitter` action
 -   `id`, `type`, `name`, `value`, `ìsPattern` for `update` action
 
+Value of `update` action and text of `post` action are expanded to numerical or boolean values instead of string values when is possible.
```

Comment:
Not sure what do you mean... In the case of update:

```
{
    "name": "blood_rule_update",
    "text": "select *,\"blood_rule_update\" as ruleName, *, ev.BloodPressure? as Pressure from pattern [every ev=iotEvent(BloodPressure? > 1.5 and type=\"BloodMeter\")]",
    "action": {
        "type": "update",
        "filter": {
            "type": "SensorMetter",
            "q": "status:on"
        },
        "parameters": {
            "id": "${id}_example",
            "version": 2,
            "attributes": [
                {
                    "name": "pressure",
                    "type": "Number",
                    "value": "${Pressure}"
                }
            ]
        }
    }
}
```

Do you refer to the "value" insise the elements of the "attributes" array?

In the case of post:

```
 "action":{
      "type":"post",
      "template":"BloodPressure is ${BloodPressure}",
      "parameters":{
         "url": "http://localhost:9182/${type}/${id}",
         "method": "PUT",
         "headers": {
            "Content-type": "text/plain",
            "X-${type}-pressure": "${BloodPressure}"
         },
         "qs": {
            "${id}": "${BloodPressure}"
         }
      }
   }
```

there is no field named "text"...

(We can clarify this at skype/F2F)